lib/resources/content/tsconfig.json

```diff
@@ -10,7 +10,13 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "moduleResolution": "node",
-    "lib": ["es2017", "dom"]
+    "lib": ["es2017", "dom"],
+    "baseUrl": ".",
+    "paths": {
+      "*": [
+        "./src/*"
```

Comment:
Not a typescript guru, so forgive me. Is this line making every path based from the `./src` folder (in essence the root of where files are requested)? What happens if I store my files in a folder other than `src`?